src/config.js

```diff
@@ -4,7 +4,7 @@
 
 module.exports.RETRY_DELAY = 1000;
 module.exports.LND_INIT_DELAY = 5000;
-module.exports.NOTIFICATION_DELAY = 5000;
+module.exports.NOTIFICATION_DELAY = 2000;
```

Comment:
Hmm, some of them it feels like there's not enough time to click the `handler` but others with shorter messages it feels like a good amount of time. I wish there were an easier way to have the notifications stay until the user dismisses it :thinking:

Reply:
Maybe ntfns with a handler could have a longer delay somehow?

Reply:
Good point. I retested this and 5 seconds is actually feels like a good amount and of time and should give the user enough time to register and react to the handler. So I reverted this change.